src/macros.jl

```diff
@@ -2,7 +2,7 @@ using JuMP
 import JuMP: getvalue, validmodel, addtoexpr_reorder
 using Base.Meta
 
-export @polyvariable, @polyconstraint
+export Poly, @set, @polyvariable
```

Comment:
Oh I get it now :D
It has nothing to do in this PR, it must have messed up my rebase ^^